contracts/ArenaToken.sol

```diff
@@ -147,6 +156,13 @@ contract ArenaToken is ERC20, ERC20Burnable, Ownable, ERC20Permit, ERC20Votes {
      * @param amount The quantity of tokens to mint.
      */
     function mint(address dest, uint256 amount) external onlyOwner {
+        require(
+            amount <= (totalSupply() * MINT_CAP) / 10_000,
```

Comment:
just to note: as the totalSupply will increase on every mint the number of tokens minted will be increased